app/controllers/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+  def index
+    @user = User.all
+  end
```

Comment:
- [ ] [OPTIONAL] Since you are trying to get all the Users in the database, the variable would be better named `@users` instead of `@user`.